src/operator/nn/softmax.cc

```diff
@@ -12,6 +12,27 @@ namespace op {
 DMLC_REGISTER_PARAMETER(SoftmaxParam);
 
 MXNET_OPERATOR_REGISTER_UNARY(softmax)
+.describe(R"code(Applies the Softmax function on the input array re-scaling 
+the elements such that they lie in the range (0,1) and the elements along 
+the given axis sum up to 1.
```

Comment:
Run-on sentence. Also this is inaccurate. Softmax doesn't just rescale elements (implies a linear transformation). Softmax is a nonlinear mapping. Also, "softmax" should not be capitalized.

Also double check, the range should not be open on both ends, no? For example, if the input has exactly 1 element, then it should come out as [1.0].

Change to
"Applies the softmax function. The resulting array contains elements in the range (0,1) and the elements along the given axis sum up to 1.